File: SampleProjects/GoT-Characters-master/Objective-C/GoT/NSData+JSON.m
//
//  NSData+JSON.m
//  GoT
//
//  Created by Paciej on 24/10/15.
//  Copyright © 2015 Maciej Piotrowski. All rights reserved.
//

#import "NSData+JSON.h"

@implementation NSData (JSON)

- (id)JSONObject {
    return [NSJSONSerialization JSONObjectWithData:self
                                           options:NSJSONReadingAllowFragments
                                             error:nil];
}

@end

File: SampleProjects/GoT-Characters-master/Swift/GoT/NSData+JSON.swift
//
//  NSData+JSON.swift
//  GoT
//
//  Created by Paciej on 24/10/15.
//  Copyright © 2015 Maciej Piotrowski. All rights reserved.
//

import Foundation

extension Data {
    var jsonObject: Any? {
        return try? JSONSerialization.jsonObject(with: self as Data, options: .allowFragments)
    }
}